cmd/kyma/alpha/create/module/module.go

```diff
@@ -273,12 +276,36 @@ func (cmd *command) Run(ctx context.Context) error {
 		archiveFS = memoryfs.New()
 		l.Info("using in-memory archive")
 	}
-	// this builds the archive in memory, Alternatively one can store it on disk or in temp folder
-	archive, err := module.CreateArchive(archiveFS, cmd.opts.ModuleArchivePath, modDef)
-	if err != nil {
-		cmd.CurrentStep.Failure()
-		return err
+
+	var archive *comparch.ComponentArchive
+	gitPath, err := files.SearchForTargetDirByName(modDef.Source, ".git")
+	if gitPath == "" || err != nil {
+		l.Warnf("could not find git repository root, using %s directory", modDef.Source)
+		fmt.Println()
+		nice.NewNice().PrintImportant("! CAUTION: The target folder is not a git repository. Therefore, the security scan will not be performed and the module sources will not be added to the layer")
```

Comment:
security scan config file in this step has not been detected yet, We don't know if user provide this config file, for open source users, they don't need security scan file to generate module. 

The condition to have this message is when CLI detect is not git repo **and** security scan config file exists.